### PR TITLE
73557: All IVC Forms - Upload forms and separated PDF attachments into S3

### PR DESCRIPTION
**Pull request by balexandr**
## Summary
Recently, we removed the combing of PDFs for 10-10D and separated them by attachment for S3 upload. This PR aims to get those attachments cleaning to an S3 bucket.

- Update vha_10_10d model to return file_paths instead of file_names
- Update UploadsController to handle multiple PDFs instead of just the compiled one at the end.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/73557

## Testing done
- Manual browser testing
- Test suite

## Screenshots
`vha_10_10d1.pdf` is the optional PDF
Screenshot 2024-02-29 at 2 00 21 PM

## What areas of the site does it impact?
- IVC Forms only

## Acceptance criteria
- needed for all IVC forms and attachments (including 10-10d, 10-7959f-1 and 2, and 10-7959a and c and other CHAMPVA forms
- the files need to be separated and available on PEGA S3

### CODE REVIEW

File: modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb
Comment (cloudmagic80) [inline modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb:1]: Let me download this branch and see what I get

**Review by reiting — APPROVED** [review]

File: modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb
Comment (balexandr) [reply modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb:1]: @cloudmagic80 Was this kosher on your end?

Comment (cloudmagic80) [reply modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb:1]: Looks good so far. I was able to attachment a few attachments in different sections of the UI and I can see them.

Comment (cloudmagic80) [reply modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb:1]: I also confirm that I see it on the s3 bucket (non-VA owned)
I changed the settings.yml file. What about adding the secrets to Parameter Store?

Comment (balexandr) [reply modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb:1]: @cloudmagic80 I'm looking into that here https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/77262

**Review by ryan-mcneil — APPROVED** [review]
👍

**Pull request merged**